config/locales/en.yml

```diff
@@ -76,6 +76,7 @@ en:
       - :month
       - :year
   department_created: Department was successfully created.
+  department_updated: Department was successfully updated.
   departments: Departments
```

Comment:
should there also be a department_destroyed? I get en.department_destroyed translation missing at the develop